src/server.c

```diff
@@ -5549,7 +5555,7 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                 "uptime_in_seconds:%I\r\n", (int64_t)uptime,
                 "uptime_in_days:%I\r\n", (int64_t)(uptime / (3600 * 24)),
                 "hz:%i\r\n", server.hz,
```

Comment:
i think it's better to show `clients_hz` here, since it is out of the hz control

Reply:
Not sure what you're asking/suggesting here.  I don't think we can just change the definition of the existing metric.

I think maybe you're suggesting a new metric?  Currently the required hz for the `clientsTimeProc` is computed locally to the function, in each iteration.  Are you suggesting adding a new field to `server...` and adding a metric above `hz` with `clients_hz`?

Reply:
yes, a new metric, since the `clients_hz` can be changed and affect the poll_wait's sleep time, this will further affect the CPU usage

Reply:
done